spec/STORE.md

```diff
@@ -113,20 +113,18 @@ packages:
 
 ## `.modules.yaml`
 
-In order to disallow the usage of different stores from the same node_modules, the `.modules.yaml` file contains information about the origin of packages. This file is required only in node_modules that are outside of the store and has to be in the root of the node_modules folder.
-
-> Can you explain what you mean? I don't get it
-
-> zkochan: Lets say you have two stores: X and Y. And you have a package - A. You installed the dependencies of A using the X store. Now you don't want to install other dependencies using store Y. You want all the dependencies in A's node_modules to be from the same store.
+A file in the root of node_modules with meta information.
 
 ### `storePath`
 
-Has the absolute path to the store that is used for installing the dependencies.
+The absolute path to the store which is used for installing dependencies in the current node_modules.
```

Comment:
Why do we need to use the absolute path? This would mean you would have to update this file when moving the package directory around on your FS.